docs/ci_cd/01-github.md

```diff
@@ -14,6 +14,7 @@ This guide will walk you through the process of setting up a GitHub Action to bu
 Before getting started, you'll need:
 - A Massdriver account 
 - A Massdriver [service account](/platform/service-accounts)
+- A GitHub repository with your application code
```

Comment:
can you toss a link to register on line 15?

Reply:
Register what? Their github repo?

To install our GitHub action, it just gives you the step needed to install:
![image](https://github.com/massdriver-cloud/docs/assets/8037512/30a15c12-c012-45c5-ada6-42ea877ded84)

Reply:
Massdriver registration page!